src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -436,9 +452,45 @@ def mousePressEvent(self, event):
             This method is called to handle a mouse press event
             :param event: the mouse event
         """
+        print("mousePress even called with event: " + str(event))
+        if self.cursor:
+            self.removeItem(self.cursor)
+        self.isPressed = True
+        if (
+                2 * QtGui.QVector2D(event.pos() - self.rect.center()).length()
+                < self.rect.width()
+        ):
+            self.drag_position = event.pos() - self.rect.topLeft()
+        super().mousePressEvent(event)
+        x, y = linear_transform(
+            math.floor(event.scenePos().x()), math.floor(event.scenePos().y()),
+            self.rows, self.cols)
+        is_coloured = (x, y) in self.according_color_dict
+        self.is_current_pixel_coloured = is_coloured
+        self.draw_cursor(event.scenePos().x(), event.scenePos().y(),
+                         self.draw_tool_radius, new_circle=True)
+
+        if self.is_current_pixel_coloured:
+            self.fill_pixels_within_circle(event.scenePos().x(),
+                                           event.scenePos().y())
+        else:
+            self.remove_pixels_within_circle(event.scenePos().x(),
+                                             event.scenePos().y())
+        self.update()
+
+    def mousePressEvent(self, event):
```

Comment:
You have two mousePressEvent functions??

Reply:
bump, this has gotta get sorted. If you need a hand then let me know and I can have a proper look

Reply:
They are both necessary, updated doc strings to reflect their functionality